README.md

```diff
@@ -2,7 +2,7 @@
 
 For nonprofits — the simplest way to give your supporters a beautiful donation experience. This button opens a donation flow through [Every.org](https://www.every.org/nonprofits).
 
-See live example here: https://assets.every.org/donate-button-v2/
+See this in production helping raise funds at https://ffungi.org/eng/
```

Comment:
cool, but think we should still provide a demo version of it to showcase all the options - looks like the assets.every.org one is broken for some reason (split panel isn't working) so i'll look into that.